plugins/jreleaser-gradle-plugin/src/main/groovy/org/jreleaser/gradle/plugin/internal/dsl/deploy/maven/AbstractMavenDeployer.groovy

```diff
@@ -80,7 +81,7 @@ abstract class AbstractMavenDeployer implements MavenDeployer {
         authorization = objects.property(Http.Authorization).convention(Providers.<Http.Authorization> notDefined())
         stagingRepositories = objects.listProperty(String).convention(Providers.<List<String>> notDefined())
         extraProperties = objects.mapProperty(String, Object).convention(Providers.notDefined())
-
+        snapshotSupported = objects.property(Boolean).convention(false)
```

Comment:
This value must also be set on the target core element in https://github.com/jreleaser/jreleaser/blob/223db09955217537d63a2dc5c32b4489cf4bd401/plugins/jreleaser-gradle-plugin/src/main/groovy/org/jreleaser/gradle/plugin/internal/dsl/deploy/maven/AbstractMavenDeployer.groovy#L147